crates/bevy_ui/src/lib.rs

```diff
@@ -75,6 +78,60 @@ impl Default for UiScale {
     }
 }
 
+/// General representation of progress between two values.
+#[derive(Debug, Clone)]
+pub struct Progress<T>
+where
+    T: Send + Sync + Clone + 'static,
```

Comment:
Why is `T` required to be `'static` ?

Reply:
Ah, I tried deriving `Reflect`, but not sure if useful or not. Then I forgot to do proper cleanup as a result.